### PR TITLE
slider

### PR DESCRIPTION
**Pull request by Shailseen**
image

### CODE REVIEW

**Review by vishalg8454 — APPROVED** [review]
Looks good, can merge.

File: component css/slider.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    font-size: smaller;
+    color: var(--dark-seocndary-color);
+}
+
+.slider-style::-webkit-slider-thumb {
+    background: var(--primary-color); 
+    -webkit-appearance: none; 
+    appearance: none;
+    width: 15px;
+    height: 15px;
```
Comment (rajkishancode) [inline component css/slider.css]: try to use rem.

**Review by rajkishancode — APPROVED** [review]
Everything else looks good.

File: component css/slider.css (hunk 1 above)
Comment (Shailseen) [reply component css/slider.css]: ok i will do that.

**Pull request merged**